GitMeFood/ContentView.swift

```diff
@@ -8,14 +8,15 @@
 import SwiftUI
 
 struct ContentView: View {
-    
     var menuItems: [String] = [
         "Chicken Pesto Pizza",
         "Sushi",
         "BBQ Sandwich",
         "Chicken and Rice",
-        "Whipped Brie"
+        "Whipped Brie",
+        "Caesar Salad"
```

Comment:
This is a great addition

Reply:
Thank you! I just wish they didn't have anchovies in them.